aws-lambda/pygeoapi-serverless-config.yml

```diff
@@ -32,11 +32,11 @@ server:
     bind:
         host: 0.0.0.0
         port: 80
-    url: /dev/
```

Comment:
It could be I was doing this wrong, but the links were failing when relative to the /dev path.